app/controllers/super_admins/external_users_controller.rb

```diff
@@ -17,6 +17,8 @@ def new
   end
 
   def create
+    # downcase email_confirmation - devise will downcase the email
+    params[:external_user][:user_attributes][:email_confirmation].downcase!
```

Comment:
Why would we do that?  The configuration option to specify case insensitivity for this field is hard coded in config/devise.rb:44 - I don't think it can be changed mid-stream.